.github/workflows/release-nightly.yml

```diff
@@ -43,3 +44,36 @@ jobs:
           prerelease: true
           removeArtifacts: true
           tag: 'nightly'
+
+  release_pypi:
+    name: Make Release on pypi.anaconda.org
+    needs: build_wheels
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
I don't know if we need to checkout in this step? I think above it was needed because we run the `rename-dev-wheels.py` script?